omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ConceptResource1_8.java

```diff
@@ -348,10 +348,13 @@ public void remove(ConceptDescription item) {
 	 * @param instance
 	 * @param mappings
 	 */
-	@PropertySetter("mappings")
-	public static void setMappings(Concept instance, List<ConceptMap> mappings) {
-		instance.setConceptMappings(mappings);
-	}
+    @PropertySetter("mappings")
+    public static void setMappings(Concept instance, Set<ConceptMap> mappings) {

```

Comment:
This should be a list, by convention that's how all other collections are represented via rest